src/cpp/core/include/core/http/TcpIpAsyncClientSsl.hpp

```diff
@@ -224,12 +258,18 @@ class TcpIpAsyncClientSsl
          }
       }
 
+      std::vector<X509*> getCertificates() const
```

Comment:
Let's add a comment here making it extra clear that the caller should not free the pointers as they're owned by the cert store.